src/ontology/cl-edit.owl

```diff
@@ -33131,6 +33132,19 @@ AnnotationAssertion(rdfs:label obo:CL_4052027 "atretic theca cell")
 SubClassOf(obo:CL_4052027 obo:CL_4033089)
 SubClassOf(obo:CL_4052027 ObjectSomeValuesFrom(obo:RO_0002202 obo:CL_0000503))
 
+# Class: obo:CL_4052026 (type IIx muscle cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727990") Annotation(oboInOwl:hasDbXref "PMID:35908794") Annotation(oboInOwl:hasDbXref "PMID:36361732") Annotation(oboInOwl:hasDbXref "doi:10.1101/2024.05.15.594276") obo:IAO_0000115 obo:CL_4052026 "A fast type II muscle cell that is part of the skeletal muscle tissue. This cell is characterized by its intermediate metabolic profile, utilizing both glycolytic and oxidative pathways for energy production. In humans, it is distinguished by the expression of myosin heavy chain 1 (MYH1).")
+AnnotationAssertion(terms:contributor obo:CL_4052026 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052026 "2024-11-04T10:26:29Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727990") oboInOwl:hasExactSynonym obo:CL_4052026 "type 2x muscle cell")
```

Comment:
Consider adding the synonym: type 2x fiber 
from PMID: [34727990](https://pubmed.ncbi.nlm.nih.gov/34727990/)